### PR TITLE
feature: ECA-8 contact form modal window

### PR DESCRIPTION
**Pull request by valkandyba**

### CODE REVIEW

File: src/components/UI/ModalWindow/ModalWindow.tsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+};
+
+const portalElement = document.getElementById('overlays');
+
+interface ModalWindowProps {
+  onClose: () => void;
+  children: ReactNode;
+}
+
+const ModalWindow: FC<ModalWindowProps> = ({ onClose, children }) => {
```
Comment (ValeriyaZavozova) [inline src/components/UI/ModalWindow/ModalWindow.tsx]: It's cool, but you could use `Modal` from material-ui :)

File: src/components/ContactForm/ContactForm.tsx [hunk 2]
```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import ContactFormImg from '../../assets/contact-form-img.png';
+import { Button, TextField, Typography } from '@mui/material';
+import classes from './ContactForm.module.scss';
+
+const ContactForm = () => {
```
Comment (ValeriyaZavozova) [inline src/components/ContactForm/ContactForm.tsx]: Create `submit` function with `console.log()` for this form.

File: src/components/UI/ModalWindow/ModalWindow.tsx (hunk 1 above)
Comment (valkandyba) [reply src/components/UI/ModalWindow/ModalWindow.tsx]: fixed

File: src/components/UI/ModalWindow/ModalWindow.tsx [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    <div>
+      <Dialog
+        open={open}
+        onClose={handleClose}
+        aria-labelledby='alert-dialog-title'
+        aria-describedby='alert-dialog-description'
+      >
+        <DialogContent>{children}</DialogContent>
+      </Dialog>
+    </div>
```
Comment (ValeriyaZavozova) [inline src/components/UI/ModalWindow/ModalWindow.tsx]:
```suggestion
      <Dialog
        open={open}
        onClose={onClose}
        aria-labelledby='alert-dialog-title'
        aria-describedby='alert-dialog-description'
      >
        <DialogContent>{children}</DialogContent>
      </Dialog>
```

File: src/components/UI/ModalWindow/ModalWindow.tsx [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+}
+
+const ModalWindow: React.FC<ModalWindowProps> = ({
+  open,
+  onClose,
+  children,
+}) => {
+  const handleClose = () => {
+    onClose();
+  };
```
Comment (ValeriyaZavozova) [inline src/components/UI/ModalWindow/ModalWindow.tsx]: for what purpose?
```suggestion
```

**Review by ValeriyaZavozova — APPROVED** [review]

File: src/components/UI/ModalWindow/ModalWindow.tsx (hunk 3 above)
Comment (valkandyba) [reply src/components/UI/ModalWindow/ModalWindow.tsx]: fixed

File: src/components/UI/ModalWindow/ModalWindow.tsx (hunk 4 above)
Comment (valkandyba) [reply src/components/UI/ModalWindow/ModalWindow.tsx]: In order for the modal window to close on click on the submit

File: src/components/ContactForm/ContactForm.tsx (hunk 2 above)
Comment (valkandyba) [reply src/components/ContactForm/ContactForm.tsx]: fixed

File: src/components/UI/ModalWindow/ModalWindow.tsx (hunk 4 above)
Comment (valkandyba) [reply src/components/UI/ModalWindow/ModalWindow.tsx]: fixed

**Pull request merged**